.rubocop_todo.yml

```diff
@@ -95,7 +106,7 @@ Style/SingleLineBlockParams:
   Exclude:
     - 'app/labor/markdown_fixer.rb'
 
-# Offense count: 5168
+# Offense count: 534
```

Comment:
they finally set the default length to 120 :-) I'm going to enforce it in a following PR

Reply:
Biggest Offense count drop yet!